SpatialGDK/Source/SpatialGDK/Public/Utils/EngineVersionCheck.h

```diff
@@ -7,7 +7,7 @@
 // GDK Version to be updated with SPATIAL_ENGINE_VERSION
 // when breaking changes are made to the engine that requires
 // changes to the GDK to remain compatible
-#define SPATIAL_GDK_VERSION 1
+#define SPATIAL_GDK_VERSION 2
```

Comment:
Why is this only two, surely we've made a lot more versions than that, is this still used?

@m-samiec

Reply:
It's only recently been introduced.